tests/test_uniform.py

```diff
@@ -9,6 +11,7 @@ class TestUniform(unittest.TestCase):
 
     maxDiff = None
 
+    @pytest.mark.xfail
```

Comment:
Why is this marked as xfail now?

Reply:
Same as before - the code for the opengraph protocol is not correct.

Reply:
I propose the same here than I proposed here: https://github.com/scrapinghub/extruct/pull/139/files#r440913833